symbolic-common/src/path.rs

```diff
@@ -509,7 +509,19 @@ impl DSymPathExt for Path {
             parent = parent.parent()?;
         }
 
-        if parent.file_stem() == Some(framework) && parent.is_dsym_dir() {
+        // Accept both Filename.dSYM and Filename.framework.dSYM as
+        // the bundle directory name.
+        let stem_matches = parent.file_name()
+            .and_then(|name| Path::new(name).file_stem())
+            .map(|stem| {
+                if stem == framework {
+                    return true;
+                }
+                let mut alt = framework.to_owned();
```

Comment:
This is annoying. I guess in hindsight I could have done:
```rust
  let alt = Path::new(framework);
  alt.file_stem() == Some(framework) && alt.extension() == Some(".framework")
```

Reply:
Might as well change, you need to make rustfmt happy anyway :-D